flink-kubernetes/src/test/java/org/apache/flink/kubernetes/kubeclient/decorators/InitTaskManagerDecoratorTest.java

```diff
@@ -179,11 +179,24 @@ void testMainContainerPorts() {
     @Test
     void testMainContainerEnv() {
```

Comment:
This method seems to test if `ResourceManagerOptions.CONTAINERIZED_TASK_MANAGER_ENV_PREFIX` is correctly added as container env vars, I would suggest to create a separate test method to test whether node id is correctly set.

Reply:
Done